README.md

```diff
@@ -9,16 +9,22 @@ When working with this logger library, please make sure you use one of the init
 Note: You can still create your own standard logger by using the `NewLogger` function (check [Logrus](https://github.com/sirupsen/logrus/blob/master/logger.go#L69) for more details).
 
 
-### Creating an Entry
-The library maintains most of the Logrus's way of logging, but it adds some default fields (`@time`, `transaction_id` and `service_name`) for the created logging entries.
-For entry creation you can use the following methods:
-- `NewEntry` - with `transaction_id` as a parameter
-- `NewMonitoringEntry` - with `transaction_id`, `eventName` and `contentType` as parameters. Beside these, a `monitoring_event=true` field will also be added to the entry. This message will be picked up by the monitoring services and dashboards.
+### Logging a Monitoring Event
+The library maintains most of the Logrus's way of logging, but it adds some default fields 
+(`@time`, `transaction_id` and `service_name`) to facilitate monitoring of relevant application events.
+You can add a monitoring event to a log entry by using the following method:
+- `WithMonitoringEvent` - with `transaction_id`, `eventName` and `contentType` as parameters. 
+Beside these, a `monitoring_event=true` field will also be added to the entry. 
+This message will be picked up by the monitoring services and dashboards.
 
 ### Adding additional fields to the Entry
 
-Beside the With... fields offered by the original Logrus Entry, the `WithUUID`, `WithTime` and `WithValidFlag` methods can also be used.
-The validation flag should mark whether the message is accepted as valid by the service it uses. Invalid messages will be ignored by some of the monitoring statistics (SLAs).
+Beside the With... fields offered by the original Logrus Entry, the following methods can be used:
+- `WithTransactionID`, to add a transaction ID to the log entry;
+- `WithUUID`, To add a UUID to the log entry;
+- `WithTime`, to set a custom time of the logging entry; 
```

Comment:
It might be worth mentioning that this will influence the splunk log time (if that is the case)